### ISSUE TITLE
Not able to capture HTML and CSS comments into IMDONE

### ISSUE THREAD

**Issue by habitus** [imdone/imdone-atom#128]
can anyone help advise me on how to capture comments in HTML and CSS as TODOS in IMDONE?

Thanks!

**Comment by piascikj** [comment]
Can you show me an example css and html file, or snippets for each?  I want to see the syntax you are using.

**Comment by jamesgraham10** [comment]
I'm having a similar issue. Am I using the wrong syntax?

```
<template>
  <!-- TODO ensure correct data is rendering in public profile view -->
    <h1>${profile.publicName}</h1>
    <h2>${profile.location.name}</h2>
</template>

Thanks!

**Comment by piascikj** [comment]
I found the cause of this.  It seems the regex was including the `--` in the list name and it didn't pass inspection.  I changed the regex to start with `[A-Z]+` and it resolved the issue.  This fix will be in on Monday with the release of imdone.io and a major upgrade of imdone-atom.  Thanks for reporting.  Both of you!

**Comment by piascikj** [comment]
If you want to test it out, install imdone-atom-beta.  Just be sure to disable imdone-atom first.

**Comment by jamesgraham10** [comment]
Awesome! :)

**Comment by piascikj** [comment]
Fixed in imdone-atom 2.0
